src/Uno.UI/UI/Xaml/Documents/TextElement.wasm.cs

```diff
@@ -45,5 +49,18 @@ protected TextElement(string htmlTag = "span") : base(htmlTag)
 		{
 			this.SetTextDecorations(ReadLocalValue(TextDecorationsProperty));
 		}
+
+		partial void OnNameChangedPartial(string newValue)
+		{
+			if (FrameworkElementHelper.IsUiAutomationMappingEnabled)
+			{
+				AutomationProperties.SetAutomationId(this, newValue);
+			}
+
+			if (FeatureConfiguration.UIElement.AssignDOMXamlName)
+			{
+				WindowManagerInterop.SetName(HtmlId, newValue);
+			}
+		}
```

Comment:
The test `When_Padding_Is_Changed_Then_Cache_Is_Missed` captured the need of this code.

This UI Test does `app.Marked("misses")` where `misses` is a `Run` (ie, `TextElement`)

`Marked` won't find the Run without `WindowManagerInterop.SetName(...)`

Alternatively, I could just update the test to not need this, but I preferred to keep this scenario working.

Note:

https://github.com/unoplatform/uno/blob/cd03d71f1849a82e289653bbf65a069810bd272e/src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/TextBlockTests/TextBlockTests.cs#L412

The `misses1` (which failed the test before adding this code) is actually unused 😄 
We never assert it being some specific value or do anything with it.